test/e2e/system_connection_test.go

```diff
@@ -37,16 +38,21 @@ func setupConnectionsConf() {
 	os.Setenv("PODMAN_CONNECTIONS_CONF", file)
 }
 
-var systemConnectionListCmd = []string{"system", "connection", "ls", "--format", "{{.Name}} {{.URI}} {{.Identity}} {{.Default}} {{.ReadWrite}}"}
-var farmListCmd = []string{"farm", "ls", "--format", "{{.Name}} {{.Connections}} {{.Default}} {{.ReadWrite}}"}
+var (
+	systemConnectionListCmd                = []string{"system", "connection", "ls", "--format", "{{.Name}} {{.URI}} {{.Identity}} {{.Default}} {{.ReadWrite}}"}
+	systemConnectionListTLSSpecialValueCmd = []string{"system", "connection", "ls", "--format", "tls"}
+	systemConnectionListTLSCmd             = []string{"system", "connection", "ls", "--format", "{{.Name}} {{.URI}} {{.TLSCAFile}} {{.Default}} {{.ReadWrite}}"}
+	systemConnectionListmTLSCmd            = []string{"system", "connection", "ls", "--format", "{{.Name}} {{.URI}} {{.TLSCAFile}} {{.TLSCertFile}} {{.TLSKeyFile}} {{.Default}} {{.ReadWrite}}"}
```

Comment:
(I think moving the commands away from the core of the test code only impedes readability compared to including the in-line in the test, but I’ll fully defer to Podman maintainers on this.)